python/ray/serve/tests/test_config.py

```diff
@@ -196,10 +196,10 @@ def test_replica_config_default_memory_none(memory_omitted_options):
 
     if memory_omitted_options is None:
         config = ReplicaConfig("fake.import_path")
-        assert config.ray_actor_options["memory"] is None
+        assert config.ray_actor_options.get("memory") is None
```

Comment:
Now that we refactored the validation code, we can remove this unit test (`test_replica_config_default_memory_none`) altogether. It was created to make sure we passed in `None` instead of `0` as the default `memory`, so replicas could use the default memory for Ray actors.